to_production.txt

```diff
@@ -1,12 +1,21 @@
 # New
-ofl/ruwudu # https://github.com/google/fonts/pull/6549
+ofl/adlamdisplay # https://github.com/google/fonts/pull/6522
 
 # Upgrade
 ofl/bricolagegrotesque # https://github.com/google/fonts/pull/6523
 ofl/notosans # https://github.com/google/fonts/pull/5789
-ofl/notosansimperialaramaic # https://github.com/google/fonts/pull/5284
-ofl/notosansmandaic # https://github.com/google/fonts/pull/5282
+ofl/notosanshk # https://github.com/google/fonts/pull/5531
+ofl/notosansindicsiyaqnumbers # https://github.com/google/fonts/pull/5285
+ofl/notosansinscriptionalpahlavi # https://github.com/google/fonts/pull/5286
+ofl/notosanskr # https://github.com/google/fonts/pull/5532
```

Comment:
@simoncozens We're going to push the Noto *Sans* HK, KR, TC, SC updates to production

Reply:
We'll do another push for the GB18030-2022 updates.  I'm looking into how we will ship these families in Workspace editors.